docs/js/main.js

```diff
@@ -56,6 +55,22 @@ const showRecipes = function (recipes) {
     for (const recipe in recipes) {
         const recipeID = recipes[recipe].id;
         const title = recipes[recipe].name;
+        //show array of tags for each recipe
+        const allRecipeTags = recipes[recipe].tags;
+        //console.log(allRecipeTags);
+        //show each tag object
+        for (let tag in allRecipeTags) {
+            //console.log(tag, allRecipeTags[tag]);
+            let allTags = (tag, allRecipeTags[tag]);
+            let tagsArray = [];
```

Comment:
You already have an array of all the tag objects in variable allRecipeTags

Reply:
The last for...of, for...in loop is what allows you to access the key:value pairs in each tag object, and consoling it out shows each of them individually so I THOUGHT I was getting somewhere and now had the loop to work within in order to mess with a conditional with an includes method or something